linux/debian/rules

```diff
@@ -14,19 +14,6 @@ export LC_ALL=C.UTF-8
 
 include /usr/share/dpkg/pkg-info.mk
 
-# Workaround for https://bugs.debian.org/1041499 and
-# https://github.com/mesonbuild/meson/issues/12017
-# --native-file got added in Meson 0.49, so we check for that version here
-MESON_049_OR_HIGHER := $(shell expr `meson --version | sed -e 's/\.\([0-9][0-9]\)/\1/g' -e 's/\.\([0-9]\)/0\1/g' -e 's/^[0-9]\{3,4\}$$/&00/'` \>= 04900)
-
-configure_options :=
-
-ifeq ($(DEB_HOST_ARCH),$(DEB_BUILD_ARCH))
-ifeq ($(MESON_049_OR_HIGHER), 1)
-configure_options += --native-file=$(CURDIR)/debian/meson/no-exe-wrapper.ini
-endif
-endif
-
 # Unfortunately dh-python 3.20180325 (bionic) doesn't provide the virtual dh-sequence-python3
```

Comment:
Are there intermediary versions of Meson where the workaround is still needed?

Reply:
There might be if a user manually installed or compiled Meson, but I don't think we have to deal with that. 

To my knowledge there are no released mainstream-distros that ship Meson 1.2.0 (the only version affected by the bug), so I think we can remove the workaround.